Exercise_D.js

```diff
@@ -13,8 +13,22 @@
  * Consider which loop you will need to use to do this.
 Use a combination of conditionals and loops to create your answer for this question. */
 
+// function multiplyNumber(number) {
+//   CODE HERE
+// }
+
+// console.log(multiplyNumber(10));
+
 function multiplyNumber(number) {
-  // CODE HERE
+     if (number <= 0 || number === 0 || typeof number !== 'number') {
+          return 'ERROR'
+     } else if (number >= 1000000) {
+          return number 
+     } else while (number <1000000) {
+          number = number * 10 
+     }
 }
 
-console.log(multiplyNumber(10));
+console.log(multiplyNumber(0));
+console.log(multiplyNumber(99999));
+console.log(multiplyNumber(1000000));

```

Comment:
Amazing Work! This works correctly, you have made good use of the OR operator, and also to check the data is a number you have made use of the syntax type of. And to multiply the number you have made use of while which is correct.